kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/SiteEmailNotificationContent.java

```diff
@@ -80,13 +80,15 @@
     protected static FormattedText getFormattedText() {
         if (formattedText == null) {
             synchronized (LOCK) {
-                FormattedText component = (FormattedText) ComponentManager.get(FormattedText.class);
-                if (component == null) {
-                    throw new IllegalStateException("Unable to find the FormattedText using the ComponentManager");
-                } else {
-                    formattedText = component;
-                }
-            }
+		if(formattedText == null){
```

Comment:
I don't really see much of a change here since formattedText is already NPE checked. If you want to do it in the block that seems fine, just no reason to leave the other one there too.